addon/components/course-rollover.hbs

```diff
@@ -3,136 +3,144 @@
-    <h3 class="title">
-      {{t "general.courseRollover"}}
-    </h3>
-    <p class="rollover-summary">
-      {{t "general.courseRolloverSummary"}}
-    </p>
-    <div class="item title">
-      <label>
-        {{t "general.title"}}:
-      </label>
-      <Input
-        type="text"
-        @value={{this.title}}
-        @input={{fn this.addErrorDisplayFor "title"}}
-        disabled={{this.save.isRunning}}
-        placeholder={{t "general.courseTitlePlaceholder"}}
-        data-test-title
-      />
-      {{#each (await (compute this.getErrorsFor "title")) as |message|}}
-        <span class="validation-error-message">
-          {{message}}
-        </span>
-      {{/each}}
-    </div>
-    <div class="item year-select">
-      <label>
-        {{t "general.year"}}:
-      </label>
-      {{#if (is-array this.allCourses)}}
-        <select {{on "change" this.setSelectedYear}} data-test-year>
-          {{#each this.years as |year|}}
-            <option
-              disabled={{includes year this.unavailableYears}}
-              value={{year}}
-              selected={{is-equal year this.selectedYear}}
-            >
-              {{#if this.academicYearCrossesCalendarYearBoundaries}}
-                {{year}} - {{inc year}}
-              {{else}}
-                {{year}}
-              {{/if}}
-              {{#if (includes year this.unavailableYears)}}
-                ({{t "general.courseRolloverYearUnavailable"}})
-              {{/if}}
-            </option>
-          {{/each}}
-        </select>
-      {{else}}
-        <LoadingSpinner />
-      {{/if}}
-      {{#each (await (compute this.getErrorsFor "selectedYear")) as |message|}}
-        <span class="validation-error-message">
-          {{message}}
-        </span>
-      {{/each}}
+  {{#let (unique-id) as |templateId|}}
+    <div class="backtolink">
+      <LinkTo @route="course" @model={{@course}}>
+        {{t "general.backToTitle" title=@course.title}}
+      </LinkTo>
     </div>
-    <div class="advanced-options">
-      <div class="item">
-        <label>
-          {{t "general.startDate"}}:
-          <br>
-          <small>
-            ({{t "general.mustBeSameDayOfWeekAsCurrentStartDate"}})
-          </small>
+    <div class="rollover-form" {{scroll-into-view}}>
+      <h3 class="title">
+        {{t "general.courseRollover"}}
+      </h3>
+      <p class="rollover-summary">
+        {{t "general.courseRolloverSummary"}}
+      </p>
+      <div class="item title">
+        <label for="title-{{templateId}}">
+          {{t "general.title"}}:
         </label>
-        <Course::RolloverDatePicker
-          @course={{@course}}
-          @value={{this.startDate}}
-          @change={{set this.startDate}}
-        />
+        <input
+          id="title-{{templateId}}"
+          type="text"
+          value={{this.title}}
+          {{on "input" (pipe (pick "target.value" (set this.title)) (fn this.addErrorDisplayFor "title"))}}
```

Comment:
`pipe` is fine, but It think it's unnecessary here. Adding two `{{on "input"` handlers should work as well and is a bit easier to read.

Reply:
:heavy_check_mark: